web/modules/custom/joinup_news/joinup_news.module

```diff
@@ -22,3 +24,29 @@ function joinup_news_entity_bundle_field_info_alter(&$fields, EntityTypeInterfac
     }
   }
 }
+
+/**
+ * Returns the owner entity of this node if it exists.
+ *
+ * The news entity can belong to a collection or a solution, depending on
+ * how it was created. This function will return the parent of the entity.
+ *
+ * @param \Drupal\Core\Entity\EntityInterface $entity
+ *    The news content entity.
+ *
+ * @return \Drupal\rdf_entity\RdfInterface|null
+ *    The parent of the entity. This can be a collection or a solution.
+ *    If there is no parent found, return NULL.
+ */
+function joinup_news_get_parent(EntityInterface $entity) {
```

Comment:
It was but then I needed it also for a hook node access and moved it back. When og is updated and the workaround is not needed any more, then this can move back into a protected method of the class.